docs: add oathkeeper oel (#2142)
ory/docs: docs/self-hosted/oel/oathkeeper/upgrade-oathkeeper.mdx

---
id: upgrade-oathkeeper
title: Upgrading Ory Oathkeeper OEL
---

This document provides a comprehensive guide on how to upgrade your Ory Oathkeeper Enterprise License (OEL) version. Upgrading to
the latest version ensures you have the newest features, security updates, and performance improvements.

## Prerequisites

Before proceeding with the upgrade, ensure you have:

- A valid Ory Enterprise License.
- Access to the Ory Enterprise Docker Registry.
- Backup of your current configuration.

## Pick the right version

> To upgrade Ory Oathkeeper OEL, you need to know the version you are currently running. Check your Docker or Kubernetes environment
> to find the tag. You should find your tag in this list:

Reviewer: This reads weird to me. Maybe something like that:
```suggestion
To check if there are newer versions of Ory Oathkeeper OEL, check your Docker or Kubernetes environment
to find the tag you are currently running and compare against this list:
```
would be better?
